src/bin/bat/main.rs

```diff
@@ -14,7 +14,7 @@ use std::io::{BufReader, Write};
 use std::path::Path;
 use std::process;
 
-use ansi_term::Colour::Green;
+use ansi_term::Color::Green;
```

Comment:
Cool trick that you can keep the old crate name. I didn't know about it.

But long-term I think it will be confusing, so I would prefer if you didn't keep `ansi_term` around.

Reply:
yeah, I went for the minimal changes.

Reply:
done and all green.